### DISCUSSION TITLE
[v5] npm install issue

### DISCUSSION THREAD

**Discussion by laxadev** [adonisjs/core#2032] [Help]
I get this error when I run` npm install` one of my project that I just cloned from my git repo.

```
npm ERR! code ENOLOCAL
npm ERR! Could not install from "node_modules\@adonisjs\http-server\matchit@git+
https:\github.com\thetutlage\matchit.git" as it does not contain a package.json
file.
```

What could it be the problem is?

**Comment by lucashillesheim** [comment]
I saw this happen when the `npm version` that generates the package-lock.json file is different from the version you are using.

Try to use the same npm version you used to create the project. Other way is to erase your package-lock.json and your node_modules directory, and run `npm install` again.

**Comment by MadhuSaini22** [comment]
Try this command to change the npm version:
npm -g install npm@4.2.0 
just change the version you are currently using  and try again.
And check if npm is installed globally on to your device or not and if not try to install it globally :)

**Comment by koteesy** [comment]
Try delete package-lock.json
